oap-server/server-bootstrap/src/main/resources/lal/example.yaml

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rules:
```

Comment:
I am feeling whether we should keep example in the release, or doc folder only?

Reply:
I am feeling an example in the doc with explanation line by line, should be more friendly. And we don't need to include these in the dist.
Also, these files in `main/resources` would be included in the jar files, rather than real `/config` of dist.

Reply:
> I am feeling an example in the doc with explanation line by line, should be more friendly.

From the perspective of an end user, I'll consider an example configuration in dist more friendly, users tend to copy-and-modify an existing example config file, instead of composing a new config file from scratch. This example file is not activated by default b/c we have activation rules config there. However, I don't strongly insist of this if you prefer to move it into the docs

Reply:
Example is fine, but moving them to `/config-examples` seems a better idea, including the previous `alarm-settings-sample.yml` file, what do you think?

Reply:
> Example is fine, but moving them to `/config-examples` seems a better idea, including the previous `alarm-settings-sample.yml` file, what do you think?

Agree